opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -234,6 +234,7 @@ def start_span(
         links: typing.Sequence[Link] = (),
         start_time: typing.Optional[int] = None,
         set_status_on_exception: bool = True,
+        record_exception: bool = True,
```

Comment:
since the span already has a variable called "record_exception", should this be renamed to something to disambiguate? for example, "do_record_exception".